smap/sources/conf/aqi_converter_conf.ini

```diff
@@ -0,0 +1,14 @@
+[server]
+Port = 8005
+
+[/]
+uuid = 2aae1481-14b2-13e1-1a44-3703cd4c5e01
```

Comment:
I'm going to merge this change, but I just want to understand the underlying behavior here.
So I see only one UUID in the conf file.
That is the UUID for the sensor, right?

Where are the UUIDs for the sensors, and how does the driver know which sensor it is converting values for?